rust/feature-flags/src/config.rs

```diff
@@ -220,6 +220,22 @@ pub struct Config {
     #[envconfig(default = "true")]
     pub test_before_acquire: FlexBool,
 
+    // PostgreSQL statement_timeout for reader database queries (milliseconds)
+    // - Set to 0 to disable (use database default, typically unlimited)
+    // - Readers may run longer analytical queries
+    // - Recommended: 60000-300000ms (1-5 minutes) or 0 for analytics workloads
+    // - This timeout is enforced server-side and properly kills queries
+    #[envconfig(default = "0")]
+    pub reader_statement_timeout_ms: u64,
```

Comment:
Should we have a separate timeout for the persons reader pool vs the non-persons reader pool?

Reply:
We decided that YES, we do want it.